packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -205,6 +205,9 @@ const TRANSFORMS = {
       logging: obj.logging,
       name_description: obj.name_description,
       name_label: obj.name_label,
+      chipset_info: {
+        iommu: JSON.parse(obj.chipset_info.iommu),
```

Comment:
The `JSON.parse` would throw if `iommu` is `undefined`. So you can do it like this instead:
```js
chipset_info: {
  iommu: obj.chipset_info.iommu !== undefined
    ? obj.chipset_info.iommu === 'true'
    : undefined,
}
```